aspnetcore/mvc/advanced/app-parts/3.0sample1/WebAppParts/Startup.cs

```diff
@@ -25,6 +27,11 @@ public void ConfigureServices(IServiceCollection services)
             var assembly = typeof(MySharedController).Assembly;
             services.AddControllersWithViews()
                 .AddApplicationPart(assembly);
+
+            services.AddMvc().AddRazorRuntimeCompilation(); 
```

Comment:
You execute `AddMvc`. This is not recommended in dotnetcore 3.0, since it is done via `AddControllersWihViews`

Remove the `AddMvc` call and replace the `AddControllersWithViews` with:

```csharp
            services.AddControllersWithViews()
                .AddApplicationPart(assembly)
                .AddRazorRuntimeCompilation();
```

I tested it and also works.

Reply:
@rverschuur  I made your suggested changes. Can you take a quick look?